apps/portals/cancercomplexity/src/config/synapseConfigs/datasets.ts

```diff
@@ -14,6 +14,19 @@ export const datasetSchema: GenericCardSchema = {
   title: 'datasetName',
   description: 'description',
   includeCitation: true,
+  customSecondaryLabelConfig: {
+    key: 'HOW TO DOWNLOAD',
+    value: 'This file is hosted externally, follow the External Link, below',
```

Comment:
Nitpick - could extract 2 constants and reuse

Reply:
I guess I'm confused why each one is defined twice.  Could isVsible() return a Boolean and you use the key and value?  Or remove the key and value, and rename isVisible() to something like getCustomLabel()?